general/administration/reverse-proxy.md

```diff
@@ -144,7 +144,7 @@ server {
 #
 #    location / {
 #        # Proxy main Jellyfin traffic
-#        proxy_pass http://SERVER_IP_ADDRESS:8096;
+#        proxy_pass http://SERVER_IP_ADDRESS:8096/;
```

Comment:
this feels useless a bit, as it basically makes `nginx` strip leading slash and then append one in the beginning... the only effect it has is normalizing the URI before passing further, don't think we want it here